src/code_e940_len_290.c

```diff
@@ -1,11 +1,89 @@
 #include "common.h"
 
 void begin_state_init(void) {
-    D_8009A650[0] |= 8;
+    D_8009A650[0] |= 0x8;
     GAME_STATUS->loadMenuState = 3;
 }
 
+#ifdef NON_MATCHING
+void step_init_state(void) {
+    GameStatus* gameStatus = GAME_STATUS;
+    s32 i;
+
+    if (gameStatus->loadMenuState != 0) {
+        gameStatus->loadMenuState--;
+        return;
+    }
+
+    D_8009A650[0] = 0;
+    gameStatus->areaID = 0;
+    gameStatus->isBattle = 0;
+    gameStatus->prevArea = -1;
+    gameStatus->mapID = 0;
+    gameStatus->entryID = 0;
+
+    (*gGameStatusPtr)->unk_76 = 0;
```

Comment:
GAME_STATUS on these

Reply:
I don't think it makes sense to do that, since I'm really not sure if these are even correct in the first place. Using the macro feels more like a statement despite it not making a technical difference. In general I don't really like the idea of cleaning up NON_MATCHINGs in this way. I see why you said it though. Honestly, I'd prefer something with a little less commitment like "IN_PROGRESS"